scientist-softserv/harvard-curiosity
# Set timezone

**Sets Server time correctly and also fixes timestamp display**
* * *


# What does this Pull Request do?
Updates Dockerfile to have a timezone and updates the view for harvests to show the etc converted to local

# How should this be tested?

Spin up this branch.

To test server time, ssh into the spotlight container and run command date. You will see the correct time now

To see the change of timestamps look at a dashboard of any collection you have done harvests on. They will have different timestamps. If you do a new harvest you should see the correct time on that as well.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
